backend/tests/cycle.test.js

```diff
@@ -30,7 +30,7 @@ describe('GET /cycle/:cycleId', () => {
   it('should get a cycle by ID', async () => {
     const cycle = new Cycle({
       term: 'FA19',
-      chapter: 'University of Illinois at Urbana-Champaign'
+      workspaceName: 'University of Illinois at Urbana-Champaign'
```

Comment:
maybe just add Hack4Impact to the end of the string so that we're following our own example here.